rhba-dev-env/vagrant/provisioning/roles/setup_env/tasks/main.yml

```diff
@@ -0,0 +1,207 @@
+---
+- fail:
+    msg: "This role is only supported for Fedora hosts"
+  when: ansible_distribution != 'Fedora'
+
+- name: upgrade all packages
+  become: true
+  become_user: root
+  dnf:
+    name: "*"
+    state: latest
+- name: install the latest version of docker
+  become: true
+  become_user: root
+  dnf:
+    name: docker
+    state: latest
+
+- name: "docker storage configurations"
+  become: true
+  become_user: root
+  copy:
+    dest: /etc/sysconfig/docker-storage-setup
+    content: |
+          DEVS=/dev/sdb
+          VG=docker-vg
+
+- name: "docker storage setup"
+  become: true
+  become_user: root
+  command: docker-storage-setup 
+
+- name: "enable docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   enabled: yes
+
+- name: "start docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   state: started
+
+- name: ensure group "docker" exists
+  become: true
+  become_user: root
+  group:
+    name: docker
+    state: present
+
+- name: adding user vagrant to group docker
+  become: true
+  become_user: root
+  user:
+    name: vagrant
+    groups: docker
+    append: yes
+
+- name: add new docker configuration
+  become: true
+  become_user: root
+  template:
+    src: "registries.j2"
+    dest: /etc/containers/registries.conf
```

Comment:
AFAIK this file is used by podman, docker still on /etc/sysconfig/docker, no?

Reply:
@spolti You are correct, I have my registries configured in /etc/sysconfig/docker .

Reply:
@spolti @errantepiphany This is a standard used for both docker and podman and others. The /etc/sysconfig/docker  only work on docker.

Reply:
We're doing podman builds with this dev env, not docker builds?

Reply:
@errantepiphany the environment is docker ready but if you want to use podman these registry settings will work too ... that's what i meant.